etc/spack/defaults/packages.yaml

```diff
@@ -1,33 +0,0 @@
-# -------------------------------------------------------------------------
-# This file controls default concretization preferences for Spack.
-#
-# Settings here are versioned with Spack and are intended to provide
-# sensible defaults out of the box. Spack maintainers should edit this
-# file to keep it current.
-#
-# Users can override these settings by editing the following files.
-#
-# Per-spack-instance settings (overrides defaults):
-#   $SPACK_ROOT/etc/spack/packages.yaml
-#
-# Per-user settings (overrides default and site settings):
-#   ~/.spack/packages.yaml
-# -------------------------------------------------------------------------
-packages:
-  all:
-    compiler: [gcc, intel, pgi, clang, xl, nag]
```

Comment:
oh, sorry, I must have something wrong in my config .. I thought I had submitted the pull request to NERSC/spack not LLNL/spack (site-local change) I'll cancel this PR and try again